livre/eq/render/RayCastRenderer.cpp

```diff
@@ -156,6 +168,7 @@ struct RayCastRenderer::Impl
         _nSamplesPerRay = frameData.getVRParameters().getSamplesPerRay();
         _computedSamplesPerRay = _nSamplesPerRay;
         _nSamplesPerPixel = frameData.getVRParameters().getSamplesPerPixel();
+        _drawAxis = frameData.getRenderSettings().getDrawAxis( );
```

Comment:
Just a comment, i.e., don't change: This is useless dynamic state. Better design would be to pass the frameData state to the method in need of it, instead of duplicating it here.